Components/List/List.css

```diff
@@ -0,0 +1,12 @@
+.spaced-list-style{
+    /* text-decoration: none; */
+    display: inline;
+    padding: 3rem 8rem;
```

Comment:
If you use these spacings regularly, then you can define a global variable. Otherwise, it is fine.

Reply:
this was unique combination applied for first time else i have define variables for common combination